src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -9089,53 +9089,48 @@ void VRFTerminalUnitEquipment::ControlVRFToLoad(EnergyPlusData &state,
                 this->SuppHeatingCoilLoad = SuppHeatCoilLoad;
-                if (this->DesignSuppHeatingCapacity > 0.0) {
-                    this->SuppHeatPartLoadRatio = min(1.0, SuppHeatCoilLoad / this->DesignSuppHeatingCapacity);
-                }
             } else {
+                SuppHeatCoilLoad = 0.0;
+            }
+        } else {
+            // not sure why FirstHVAC has anything to do with this but that was already here
+            // another branch should test removing FirstHVACIteration to get same answer each iteration
+            if (!FirstHVACIteration && ((QZnReq > DataHVACGlobals::SmallLoad && QZnReq > FullOutput) ||
+                                        (((QZnReq - NoCompOutput) > DataHVACGlobals::SmallLoad) && QZnReq <= 0.0))) {
+                Real64 ZoneLoad = 0.0;
+                Real64 LoadToHeatingSP = 0.0;
+                Real64 LoadToCoolingSP = 0.0;
                 getVRFTUZoneLoad(state, VRFTUNum, ZoneLoad, LoadToHeatingSP, LoadToCoolingSP, false);
-                if (((FullOutput < (LoadToHeatingSP - DataHVACGlobals::SmallLoad) ||
-                      ((QZnReq - NoCompOutput) > DataHVACGlobals::SmallLoad && QZnReq <= 0.0))) &&
-                    !FirstHVACIteration) {
-                    if ((QZnReq - NoCompOutput) > DataHVACGlobals::SmallLoad && QZnReq <= 0.0) {
-                        if (LoadToHeatingSP < 0.0 && QZnReq == 0.0) {
-                            SuppHeatCoilLoad = max(0.0, LoadToHeatingSP - FullOutput);
-                        } else {
-                            SuppHeatCoilLoad = max(0.0, QZnReq - FullOutput);
-                        }
+                if ((QZnReq - NoCompOutput) > DataHVACGlobals::SmallLoad && QZnReq <= 0.0) {
+                    if (LoadToHeatingSP < 0.0 && QZnReq == 0.0) {
+                        SuppHeatCoilLoad = max(0.0, LoadToHeatingSP - FullOutput);
                     } else {
-                        if (QZnReq > 0.0 && (NoCompOutput - QZnReq) >= DataHVACGlobals::SmallLoad) {
-                            SuppHeatCoilLoad = 0.0;
-                        } else {
-                            SuppHeatCoilLoad = max(0.0, LoadToHeatingSP - FullOutput);
-                        }
+                        SuppHeatCoilLoad = max(0.0, QZnReq - FullOutput);
                     }
-                    this->SuppHeatingCoilLoad = SuppHeatCoilLoad;
-                    if (this->DesignSuppHeatingCapacity > 0.0) {
-                        this->SuppHeatPartLoadRatio = min(1.0, SuppHeatCoilLoad / this->DesignSuppHeatingCapacity);
+                } else if (FullOutput < (LoadToHeatingSP - DataHVACGlobals::SmallLoad) && LoadToHeatingSP > 0.0) {
+                    if (QZnReq > 0.0 && (NoCompOutput - QZnReq) >= DataHVACGlobals::SmallLoad) {
+                        SuppHeatCoilLoad = 0.0;
+                    } else {
+                        SuppHeatCoilLoad = max(0.0, LoadToHeatingSP - FullOutput);
```

Comment:
In the above at line 9119, the supp heater was tuning on fully when there were small loads because the cooling coil was active. Adding ` && LoadToHeatingSP > 0.0` to indicate a valid heating load avoided this problem.